Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -977,6 +977,24 @@ public final class PackageBuilder {
                     decl = .LINK_FRAMEWORKS
                 }
 
+            case .interoperabilityMode(let lang, let version):
+                switch setting.tool {
+                case .c, .cxx, .linker:
+                    throw InternalError("only Swift supports interoperability")
+
+                case .swift:
+                    decl = .OTHER_SWIFT_FLAGS
+                }
+
+                if lang == .Cxx {
+                    guard let version else {
+                        throw InternalError("C++ interoperability requires a version")
```

Comment:
Could this error string be expanded, what version is it referring to? Is that a version of Swift or C++ standard?

Reply:
> This is the compatibility version of Swift/C++ interop, it is meant to preserve source compatibility for user projects while interop is evolving. At the moment the only supported interop version is swift-5.9 which is aligned with the version of Swift itself, but this might not always be the case in the future.

Could this paragraph be added as a doc comment or just plain comment in the code to provide context right there, which could be helpful in the future when coming back to it?

Reply:
Perfect, thanks!